.github/workflows/mpet-docker-test.yml

```diff
@@ -9,9 +9,18 @@ jobs:
     
     runs-on: ubuntu-latest
     container:
-      image: vikko00/mpet-deps:latest
+      image: vikko00/daetools:1.9.0
 
     steps:
+    - name: update container
+      run: |
+        apt-get update
+        apt-get upgrade -y
```

Comment:
Seems like running apt-get could be slow. Is this necessary?

Reply:
It's not that slow on github, only 17 seconds. (on bitbucket it takes much longer, I dont know why...)  I think it is good to keep the dependencies up to date. debian (the underlying distro) does not push new major versions of packages on their stable branches.